src/System.Text.Json/tests/Serialization/Stream.WriteTests.cs

```diff
@@ -258,6 +263,636 @@ public static async Task LargeJsonFile(int bufferSize)
                 Assert.Equal(SessionResponseCount, deserializedList.Count);
             }
         }
+
+        [Theory]
+        [InlineData(1, true, true)]
+        [InlineData(1, true, false)]
+        [InlineData(1, false, true)]
+        [InlineData(1, false, false)]
+        [InlineData(10, true, false)]
+        [InlineData(10, false, false)]
```

Comment:
Is there a reason why we certain true/false combinations are chosen based on the payload size?

Reply:
At first I was testing with all combinations, test became slower and producing very big json in writeIndented=true case, so decided to not use all combinations of options, especially for the bigger the payloads. Thought as writing without indentation works testing with indentation not much useful

Reply:
I see. How resource intensive are these new tests? And how long does it take to run them? If it is more than a couple of seconds, we should consider moving them to outerloop.

Also, do we need the ignore null option? It doesn't look like we have any nulls within the input object. If there is no difference between the null behaviors, it not necessary to test those combinations here.

Reply:
> How resource intensive are these new tests? And how long does it take to run them? If it is more than a few of seconds, we should consider moving them to outerloop. I believe the entire set of JSON tests take ~15-20 seconds on my dev box, currently.

It became 20-22 seconds, so I decreased some tests test data including the above test. So now it is runs around 18 seconds on my machine, before adding these tests it was 16-17 seconds.

> Also, do we need the ignore null option? It doesn't look like we have any nulls within the input object. If there is no difference between the null behaviors for the objects being serialized/deserialized, its not necessary to test those combinations here.

There is null values, I just didn't explicitly set them into null, there were difference with running ignore null